Makefile

```diff
@@ -15,9 +15,8 @@ GIT_REPO=metal-go
 PACKAGE_PREFIX=metal
 PACKAGE_MAJOR=v1
 
-# Pull in custom generator jar so I can use unmerged bugfixes
-OPENAPI_GENERATOR=/Users/ctreatman/Documents/code/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar
-SWAGGER=docker run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local -v $(OPENAPI_GENERATOR):/opt/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar ${IMAGE}
+# Pull in custom-built generator jar so we can use unmerged bugfixes
+SWAGGER=docker run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local -v $(CURDIR)/lib/openapi-generator-cli.jar:/opt/openapi-generator/modules/openapi-generator-cli/target/openapi-generator-cli.jar ${IMAGE}
```

Comment:
> The openapi-generator fork/branch I am using for local generation is [ctreatma/openapi-generator:local-generator-testing](https://github.com/ctreatma/openapi-generator/tree/local-generator-testing).

Let's note the source here for now. 

Perhaps we can build the branch rather than rely on the jar artifact, as in https://github.com/equinix-labs/metal-java/blob/main/Makefile#L36-L43.